src/components/dialog/GestureCheatsheet.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { css } from '../../../styled-system/css'
+import { closeDialogActionCreator } from '../../actions/closeDialog'
+import { FADEOUT_DURATION } from '../../constants'
+import CommandGrid from '../CommandGrid'
+import Dialog from './Dialog'
+import DialogContent from './DialogContent'
+import DialogTitle from './DialogTitle'
+
+/**
+ * Gesture cheatsheet component.
+ */
+const GestureCheatsheet: React.FC = () => {
+  const dispatch = useDispatch()
+  const isOpen = useSelector(state => state.dialogOpen)
+  const dialogRef = useRef<HTMLDivElement | null>(null)
+  const bottomRef = useRef<HTMLDivElement | null>(null)
+  const [isBottomVisible, setIsBottomVisible] = useState(false)
+
+  /**
+   * Handles the closure of the gesture cheatsheet.
+   */
+  const handleClose = () => {
+    if (dialogRef.current) {
+      dialogRef.current.style.opacity = '0'
+      setTimeout(() => {
+        dispatch(closeDialogActionCreator())
+      }, FADEOUT_DURATION)
+    }
+  }
+
+  useEffect(() => {
+    if (dialogRef.current) {
+      dialogRef.current.style.opacity = '1'
+    }
+  }, [isOpen])
+
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflow = 'hidden'
+    } else {
+      document.body.style.overflow = 'auto'
+    }
+    return () => {
+      document.body.style.overflow = 'auto'
+    }
+  }, [isOpen])
+
+  useEffect(() => {
+    if (!bottomRef.current || !isOpen) return
+
+    const observer = new IntersectionObserver(
```

Comment:
A simpler solution is to create 100px of padding on the bottom below all the gestures. The gradient won't be visible over the empty space. 

This would create a better user experience also since the empty space signifies to the user that they are at the end.